spec-tests/src/main/java/org/candlepin/spec/bootstrap/client/ExternalProperties.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.bootstrap.client;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.function.Supplier;
+
+/**
+ * Class for loading spec test configuration from external config file.
+ * <p>
+ * The file is expected in the home directory.
+ */
+public class ExternalProperties implements Supplier<Properties> {
+
+    private static final Path CONFIG_PATH = Paths.get(
+        System.getProperty("user.home"), ".candlepin_spec.conf");
+
+    @Override
+    public Properties get() {
```

Comment:
What do you mean by current directory? /etc/candlepin?

Reply:
Yeah. In general I run from the command line from within a given VM. If it looked at the CWD, then a candlepin_spec.conf in the candlepin repo would get picked up no matter which directory I'm in (and can be ignored by .gitignore if committing is a concern). Otherwise with the dotfiles only method from the user's root, I would have to maintain upwards of six of these if I had any customizations to make to it, or add a separate ansible task to map another directory or file in.

Reply:
Done. CWD is now checked as well.

Reply:
:+1: